SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Base.cpp

```diff
@@ -68,10 +67,15 @@ Base::Base()
     f_tags.setAutoLink(false);
     f_bbox.setOwnerClass("Base");
     f_bbox.setReadOnly(true);
-    f_bbox.setPersistent(false);
```

Comment:
The user of the persistent flag in sofa should be application-specific: in runsofa2 for instance it is used to tell the program that a data has been modified by the user and this would be saved whith the scene